### ISSUE TITLE
Please update Kodi-addon-xvdr

### ISSUE THREAD

**Issue by Karl-opec** [OpenELEC/OpenELEC.tv#4027]
Please update kodi-addon-xvdr from 2bf2563 to latest master
for oe-master and oe-5.0.

**Comment by fritsch** [comment]
Why?

No stable upgrades without a reason.

**Comment by stefansaraev** [comment]
986cf69126b3ecb3f4bae44314ee7177fcd486d9

I am NOT pushing it to 5.0

**Comment by sraue** [comment]
do we still have a real need for XVDR? whats the benefits still (and if there are some, can this be included in VNSI?) i would deprecate XVDR in favour of VNSI if possible

**Comment by stefansaraev** [comment]
I, personaly, dont need it at all. but reading forums, lot of people still use it. no idea why

**Comment by Karl-opec** [comment]
For me channel switching is quit noticeable faster as with vnsi.

**Comment by stefansaraev** [comment]
@Karl-opec vnsi has option for "pmt timeout". set it to 0 to have xvdr behaviour.

**Comment by the-dreamer** [comment]
i am using it also. When i started with openelec 4.0/vdr first time xvdr was in my opinion more stable and didn't crash on x86_64 and rpi. VNSI did. XVDR didn't show any lags and switching channel was also fast. After editing the channels.conf first time the channel list is now updated/filtered by a script. And this filtering is my main reason why i didn't switch to vnsi. I have added a feature request to the developer of vnsi to get a script support but until now it is not included. 
And yes i know that vnsi has also some filter option but it needs time to time updated to get the channels you want. I have more than 4000 channels and take to much time for me to do it.
